lib/block-patterns.php

```diff
@@ -236,6 +236,58 @@ function load_remote_patterns() {
 	}
 }
 
+/**
+ * Register featured patterns from wordpress.org/patterns.
+ *
+ * @since 5.9.0
+ */
+function load_remote_featured_patterns() {
+	/**
+	 * Filter to disable remote block patterns.
+	 *
+	 * @since 5.8.0
+	 *
+	 * @param bool $should_load_remote
+	 */
+	$should_load_remote = apply_filters( 'should_load_remote_block_patterns', true );
```

Comment:
I think it's okay to use the same flag for loading remote patterns. WDYT?

Reply:
I think it's better to use a single filter for disabling remote patterns.